packages/open-ui/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { MarginStyleKeys } from 'interfaces/MarginStyleKeys';
+import { getMarginProps } from 'utils/helpers';
+
+import styles from './Tag.module.scss';
+
+interface TagProps {
+  name: string;
+  size?: 'small' | 'large';
+}
+
+export const Tag: React.FC<TagProps & MarginStyleKeys> = props => {
+  const { name, size = 'small' } = props;
```

Comment:
I dont think we need to set a default here as `small` is the default style anyway